Multisig/UI/Transaction/ExecuteTransaction/ExecutionOptionsCellBuilder.swift

```diff
@@ -0,0 +1,185 @@
+        let cell = tableView.dequeueCell(DisclosureWithContentCell.self)
+        cell.setText("Estimated fee")
+        cell.setBackgroundColor(.backgroundSecondary)
+
+        switch model {
+        case .loading:
+            let content = loadingView()
+            cell.setContent(content)
+
+        case .empty:
+            let content = textView("Not set")
+            cell.setContent(content)
+
+        case .loaded(let feeModel):
+            let amountPiece = AmountAndValuePiece()
+            amountPiece.setAmount(feeModel.tokenAmount)
+            amountPiece.setFiatAmount(feeModel.fiatAmount)
+            cell.setContent(amountPiece)
+        }
+
+        return cell
+    }
+
+    func textView(_ text: String?) -> UIView {
+        let label = UILabel()
+        label.textAlignment = .right
+        label.setStyle(.body)
+        label.text = text
+        return label
+    }
+
+    func loadingView() -> UIView {
+        let skeleton = UILabel()
+        skeleton.textAlignment = .right
+        skeleton.isSkeletonable = true
+        skeleton.skeletonTextLineHeight = .fixed(25)
+        skeleton.showSkeleton(delay: 0.2)
+        return skeleton
+    }
+
+    func buildRelayerPayment(_ model: ExecutionOptionsUIModel, tableView: UITableView) -> UITableViewCell{
+        let cell = tableView.dequeueCell(PaymentMethodCell.self)
+        if case let .filled(relayerInfo) = model.relayerState {
+            cell.setRelaying(relayerInfo.remainingRelays, relayerInfo.limit)
+        }
+        cell.setBackgroundColor(.backgroundPrimary)
+        return cell
+    }
+
+    func buildAccountPayment(tableView: UITableView) -> UITableViewCell {
+        let cell = tableView.dequeueCell(SecondaryDetailDisclosureCell.self)
+        cell.setText("With an owner key", hideDisclousre: !chain.isSupported(feature: .relay))
+        cell.selectionStyle = chain.isSupported(feature: .relay) ? .default : .none
+        cell.setBackgroundColor(.backgroundPrimary)
+        return cell
+    }
+
+    func buildExecutedWithAccount(_ model: ExecuteWithAccountCellState, tableView: UITableView) -> UITableViewCell {
+        let cell = tableView.dequeueCell(DisclosureWithContentCell.self)
+        cell.setText("Select key")
+        cell.setBackgroundColor(.backgroundPrimary)
+        switch model {
+        case .none:
+            preconditionFailure("Developer error: CellState not properly initialized")
+
+        case .loading:
+            let content = loadingView()
+            cell.setContent(content)
+
+        case .empty:
+            let content = textView("Not selected")
+            cell.setContent(content)
+
+        case .filled(let accountModel):
+            let content = MiniAccountAndBalancePiece()
+            content.setModel(accountModel)
+            cell.setContent(content)
+        }
+        return cell
+    }
+
+    // TODO possible duplication between ReviewExecutionCellBUilder and here
+    func buildSpacing() {
```

Comment:
This is already part of TableView+Cells.swift, please evaluate if you can use it from there